src/massdistribution/Cordes.cpp

```diff
@@ -0,0 +1,50 @@
+#include "crpropa/massdistribution/Cordes.h"
+
+namespace crpropa {
+
+
+
+double Cordes::getHIIDensity(const Vector3d &position) const {
+	
+	
+	double n=0;
+	
+	double z=position.z;
+	double R = sqrt(position.x*position.x+position.y*position.y);	//radius in galactic disk
+	
+	n += 0.025*exp(-fabs(z)/(1*kpc))*exp(-pow(R/(20*kpc),2));	//galactocentric component
+	n += 0.2*exp(-fabs(z)/(0.15*kpc))*exp(-pow((R-4*kpc)/(2*kpc),2));	//anular component
+
+	return n/ccm;
```

Comment:
I find this as undesirable behaviour. If we use internally SI units everywhere, why this explicit conversion of units? Can it be avoided?